### PR TITLE
15 User Sees GitHub Followers

### PR DESCRIPTION
**Pull request by alexander-mathieu**
This PR adds followers to the GitHub API on the User's dashboard. Feature spec has been added, as well as a spec for additional functionality added to the GitHub service. Additionally:

- Adds followers method to UserShowFacade
- Adds followers_by_user to GitHub service
- github_user_repos casette has been changed to github_dashboard casette
- Dashboard casette is now required throughout all feature tests for consistency

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -12,13 +12,24 @@
...
+          </ul>
+      </section>
+      <section class='github-dashboard-followers'>
+        <h2>Followers</h2>
+          <ul>
+          <% facade.followers.each do |follower| %>
+            <li><%= link_to follower.name, follower.html_url %></li>
+          <% end %>
+          </ul>
+      </section>
```
Comment (BrennanAyers) [inline app/views/users/show.html.erb:32]: Much cleaner, and will help us in the next ticket.

File: fixtures/github_followers_by_user.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+[
+    {
+        "login": "patrickshobe",
```
Comment (BrennanAyers) [inline fixtures/github_followers_by_user.json:3]: Patrickshobe!

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -14,6 +14,7 @@
   config.hook_into :webmock
   config.configure_rspec_metadata!
   config.filter_sensitive_data("<YOUTUBE_API_KEY>") { ENV['YOUTUBE_API_KEY'] }
+  config.filter_sensitive_data('<GITHUB-TOKEN>') { ENV['GITHUB-TOKEN'] }
```
Comment (BrennanAyers) [inline spec/rails_helper.rb:17]: 👍

**Review by bplantico — APPROVED** [review]
Very similar method to Github Repositories methods/implementation so following that convention makes a lot of sense.

Refactoring the cassettes and the specs to consolidate for both Github repositories and Github followers makes a lot of sense too, plus masking the token was necessary.

**Pull request merged**